crud/upsert_many.lua

```diff
@@ -26,6 +26,7 @@ local function upsert_many_on_storage(space_name, tuples, operations, opts)
         sharding_key_hash = '?number',
```

Comment:
Why do we ignore upsert in this patch? It should be supported too

Reply:
Done, implemented